Regular-Question-Answers/find_the_town_judge.cpp

```diff
@@ -0,0 +1,93 @@
+// Author: Novruz Amirov
+// Question Link: https://leetcode.com/problems/find-the-town-judge/
```

Comment:
Comments are generally nice, but the code format should be corrected

1. Use spaces in necessary places
2. Use camelcase
3. Careful about naming variables